substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -1536,6 +1542,19 @@ pub mod env {
 		)?)
 	}
 
+	/// Returns the chain ID.
+	/// See [`pallet_revive_uapi::HostFn::chain_id`].
+	#[api_version(0)]
+	fn chain_id(&mut self, memory: &mut M, out_ptr: u32) -> Result<(), TrapReason> {
+		Ok(self.write_fixed_sandbox_output(
```

Comment:
Doesn't charge any gas except the syscall overhead. Needs a benchmark.

Reply:
Ah yes good catch. Since this doesn't really do anything except writing a constant to contract memory and the syscall overhead is apparently already charged, would the `RuntimeCosts::CopyToContract(len)` do it instead of a dedicated benchmark?

Reply:
Yeah right I see. Just added the gas token for now, thanks!